UI/package.json

```diff
@@ -27,16 +27,16 @@
   },
   "homepage": "https://github.com/speeduino/Ardu-Stim",
   "devDependencies": {
-    "electron": "^11.2.0",
-    "electron-builder": "^22.9.1",
-    "electron-packager": "^15.2.0",
-    "electron-rebuild": "^2.3.4"
+    "electron": "^16.0.7",
+    "electron-builder": "^22.14.5",
+    "electron-packager": "^15.4.0",
+    "electron-rebuild": "^3.2.7"
   },
   "dependencies": {
-    "electron-dl": "^1.14.0",
-    "request": "^2.88.0",
-    "serialport": "^9.0.4",
-    "usb": "^1.6.5"
+    "cross-fetch": "^3.1.4",
```

Comment:
Is there a reason for pulling extra library instead of using browser native [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)? (see my PR for more context: https://github.com/speeduino/SpeedyLoader/pull/25)

Reply:
To be honest I was unaware of fetch at the time of writing the code. Seeing as the current code is working I'd rather move on to different issues.

Reply:
I just replaced cross-fetch with fetch. Thanks for the comment.